.github/workflows/ci.yml

```diff
@@ -9,14 +9,15 @@ jobs:
     runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, "3.10", 3.11, pypy2.7, pypy3.9]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9, "3.10", 3.11, 3.12, pypy2.7, pypy3.9]
```

Comment:
Should this be ``3.12`` or ``3.12-dev``? I've seen both used.

Reply:
Using 3.12 and `allow-prereleases: true` will use the latest 3.12 build (3.12b4 at the time of writing this comment).

3.12-dev might also work.